UsbAccess/UsbAccess.cpp

```diff
@@ -4,16 +4,17 @@
 #include <sys/stat.h>
 #include <dirent.h>
 #include <unistd.h>
+#include <mntent.h>
 
 const short WPEFramework::Plugin::UsbAccess::API_VERSION_NUMBER_MAJOR = 1;
 const short WPEFramework::Plugin::UsbAccess::API_VERSION_NUMBER_MINOR = 0;
 const string WPEFramework::Plugin::UsbAccess::SERVICE_NAME = "org.rdk.UsbAccess";
 const string WPEFramework::Plugin::UsbAccess::METHOD_GET_FILE_LIST = "getFileList";
 const string WPEFramework::Plugin::UsbAccess::METHOD_CREATE_LINK = "createLink";
 const string WPEFramework::Plugin::UsbAccess::METHOD_CLEAR_LINK = "clearLink";
-const string WPEFramework::Plugin::UsbAccess::USB_MOUNT_PATH = "/run/media/sda"; // platco
-const string WPEFramework::Plugin::UsbAccess::LINK_URL_HTTP = "http://localhost:50050/usbdrive";
-const string WPEFramework::Plugin::UsbAccess::LIGHTTPD_CONF_PATH = "/etc/lighttpd/lighttpd.conf";
+const string WPEFramework::Plugin::UsbAccess::LINK_URL_HTTP = "http://127.0.0.1:50050/usbdrive";
```

Comment:
Is there a risk using 127.0.0.1 instead of localhost?

Reply:
reverted to "localhost"